webpack.config.js

```diff
@@ -57,7 +49,8 @@ module.exports = [
       library: 'contentful'
     },
     module: {
-      loaders
+      loaders,
+      noParse: /\/clone\//
```

Comment:
Avoids bundling of buffer polyfill with clone since we do not need buffer support an it is like 20kb big